### ISSUE TITLE
Where clause gets completely overruled by search

### ISSUE THREAD

**Issue by Jossnaz** [pods-framework/pods#3391]
I have this code

`PodsData.php` on line 643  around this code:

```
        if ( empty( $results ) ) {
            // Build
            $this->sql = $this->build( $params );

            // Debug purposes
            if ( ( 1 == pods_var( 'pods_debug_sql', 'get', 0 ) || 1 == pods_var( 'pods_debug_sql_all', 'get', 0 ) ) && pods_is_admin( array( 'pods' ) ) )
```

I print out `$this->sql`.

the sql results in:

```
                SELECT

                DISTINCT
                `t`.*
                FROM `wf_pods_volunteers` AS `t`

                    LEFT JOIN `@wp_podsrel` AS `rel_author` ON
                        `rel_author`.`field_id` = 317
                        AND `rel_author`.`item_id` = `t`.`id`

                    LEFT JOIN `wf_users` AS `author` ON
                        `author`.`ID` = `rel_author`.`related_item_id`

                WHERE ( `author`.`ID` = 22 ) AND `t`.`gender` LIKE '%Chloé%' OR `t`.`firstname` LIKE '%Chloé%' OR `t`.`lastname` LIKE '%Chloé%' OR `t`.`email` LIKE '%Chloé%' OR `t`.`skype` LIKE '%Chloé%' OR `t`.`address` LIKE '%Chloé%' OR `t`.`postcode` LIKE '%Chloé%' OR `t`.`city` LIKE '%Chloé%' OR `t`.`comment` LIKE '%Chloé%'


                ORDER BY `t`.`firstname`, `t`.`id`
                LIMIT 0, 25
```

as you can see, the restriction of the where, namely: 

 ( `author`.`ID` = 22 )  will not work, as there are no brackets around the OR statements.
e.g. this is the equivalent  to `(authorid = 1 and gender like %chloe%) or firstname like %chloe%` I mean of course, this won't work.

"Why do you need this at all?"

i want to restrict the search results to only show results that were created by the user who is doing the search.  Yes, _search results_, not just listing the items.

"How did you create this?"

using your "where" parameter in `pods_ui`

snippet:

```
    $userid = get_current_user_id();

        //adding few basic parameters
        $object->ui = array(
            'item'   => 'volunteer',
            'items'  => 'volunteers',
            'fields' => array(
                /*'add'       => $fields,
                'edit'      => $fields,
                'duplicate' => $fields,*/
                'export'    => $export_fields,
                'manage'    => $manage_fields
            ),
            //Enable the Reordering interface 
            'reorder' => array(
                //the column name to be reordered (works best if it's a Number column type)
                'on'      => 'created',
                //sort column name on reordering screen
                'orderby' => 'created desc',
            ),
            //An array of actions to disable, if you supply 'add' and/or 'edit' it will also disable the link on the Manage screens
            'actions_disabled' => $actions_disabled,

            //An array of actions to hide, if you supply 'add' and/or 'edit' it will also hide the link on the Manage screens
            'actions_hidden' => $actions_hidden,

            /*fields we want to filter*/
//            'filters' => array(
//                'book_author', 'book_genre', 'created'
//            ),
            //we need to enable enhanced filters
//           'filters_enhanced' => true,
/*
            //views are not working fully yet but lets implement them just to give you basic idea what they can be used for
            'views' => array(
                'newest' => 'Show recent ones',
                'no-stock' => 'Out of stock',
            )*/
//            'restrict' => array(
//                'author_restrict' => 'author.ID'
//            ),
//            'where' => 'author.ID = ' . (int) $userid,
          'where' => array("manage" => 'author.ID = ' . (int) $userid),
          'sql' => "lukassql"



            /* Restrict example based on https://github.com/pods-framework/pods/issues/623#issuecomment-12730649
            'edit' => array(
                'relation' => 'AND', // default is AND
                // Yes/No field
                'author.ID' => $userID 
            ),*/

        );         

        pods_ui($object, false);
```

**Comment by pglewis** [comment]
@sc0ttkclark: Do you happen to know a work-around for this off-hand?

**Comment by sc0ttkclark** [comment]
What version Pods are you running @Jossnaz?

**Comment by sc0ttkclark** [comment]
@pglewis: https://github.com/pods-framework/pods/blob/2.x/classes/PodsData.php#L1075-L1076

We should be using ( ) around any OR clauses, so this seems weird.

**Comment by sc0ttkclark** [comment]
This is the only other case I can think of that doesn't use ( ) around an OR clause, but it's in field() https://github.com/pods-framework/pods/blob/2.x/classes/Pods.php#L1250-L1252

**Comment by Jossnaz** [comment]
@sc0ttkclark  we run  2.3.18

**Comment by sc0ttkclark** [comment]
I bet this was fixed in 2.4, 2.5, or 2.6, you should really try upgrading to the latest Pods 2.6.1 (out today).

**Comment by quasel** [comment]
Hi, we didn't hear back from you - so I'm closing the issue!
